### DISCUSSION TITLE
Book Recommendations

### DISCUSSION THREAD

**Discussion by Gramarin** [frePPLe/frepple#594] [General]
What are some good books that will take my understanding of frePPLe to the next level? More towards operations management rather than software development focused.

**Comment by jdetaeye** [comment]
I can't recommend individual titles, but here are some good categories to look for...

For the business process side of frepple:
- Supply chain management:  https://www.amazon.com/s?k=Supply+Chain+Management&rh=n%3A283155 
- Inventory management: https://www.amazon.com/s?k=inventory+management
- Advanced planning and scheduling: https://www.amazon.com/s?k=advanced+planning+and+scheduling
- Demand forecasting: https://www.amazon.com/s?k=demand+forecasting

For the technical side of frepple:
- django:  https://www.amazon.com/s?k=django+book

**Reply by Gramarin** [reply]
Many thanks Johan!
